src/sif/motorcyclecost.cc

```diff
@@ -245,12 +246,14 @@ class MotorcycleCost : public DynamicCost {
    * @param  node  Node (intersection) where transition occurs.
    * @param  pred  the opposing current edge in the reverse tree.
    * @param  edge  the opposing predecessor in the reverse tree
+   * @param  has_measured_speed Do we have any of the measured speed types set?
    * @return  Returns the cost and time (seconds)
    */
   virtual Cost TransitionCostReverse(const uint32_t idx,
                                      const baldr::NodeInfo* node,
                                      const baldr::DirectedEdge* pred,
-                                     const baldr::DirectedEdge* edge) const override;
+                                     const baldr::DirectedEdge* edge,
+                                     const bool has_measured_speed) const override;
```

Comment:
Is `has_measured_speed` unused here?

Reply:
@mandeepsandhu we are missing RAD tests for this and it is noted as a TODO

Reply:
ignore this comment, I didn't realize this was the function declaration.